src/components/document-metadata-modal-utils.js

```diff
@@ -145,9 +145,8 @@ export function determineTargetRoomId({ mode, initialDocumentMetadata, documentT
           throw new Error(`Invalid cloning strategy: '${cloningStrategy}'`);
       }
     case DOCUMENT_METADATA_MODAL_MODE.create:
-      return initialDocumentMetadata.roomId || null;
     case DOCUMENT_METADATA_MODAL_MODE.update:
-      return null; // Not used for metadata updates
+      return initialDocumentMetadata.roomId || null;
```

Comment:
Decided to make this fully into the "single point of truth" for the room ID, regardless of what it is called for.

Reply:
Yup :+1: